values.yaml

```diff
@@ -47,3 +46,59 @@ deploy:
   nameOverride: ""
 
   resourceLabels: []
+
+  infinispan:
+    cacheContainer:
+      security:
+        authorization: ~
+      transport:
+        cluster: ${infinispan.cluster.name:cluster}
+        stack: kubernetes
+        node-name: ${infinispan.node.name:}
+    server:
+      interfaces:
+        - name: public
+          inetAddress:
+            value: ${infinispan.bind.address:127.0.0.1}
+      socketBindings:
+        defaultInterface: public
+        portOffset: ${infinispan.socket.binding.port-offset:0}
+        socketBinding:
+          - name: default
+            port: 11222
+          - name: metrics
+            port: 11223
+      security:
+        credentialStores:
+          - name: credentials
+            path: credentials.pfx
+            clearTextCredential:
+              clearText: secret
+        securityRealms:
+          - name: default
+            propertiesRealm:
+              groupsAttribute: Roles
+              userProperties:
+                path: users.properties
+              groupProperties:
+                path: groups.properties
+          - name: metrics
+            propertiesRealm:
+              groupsAttribute: Roles
+              userProperties:
+                path: metrics-users.properties
+                relativeTo: infinispan.server.config.path
+                plainText: true
+              groupProperties:
+                path: metrics-groups.properties
+                relativeTo: infinispan.server.config.path
+      endpoints:
```

Comment:
```suggestion
                relativeTo: infinispan.server.config.path
      # Do not modify the default endpoints.
      endpoints:
```

Reply:
I have moved this comment to the "metrics" endpoint and updated the message.